querydsl-sql/src/test/java/com/querydsl/sql/SelectBase.java

```diff
@@ -395,7 +395,7 @@ public void Custom_Projection() {
     }
 
     @Test
-    @IncludeIn({H2, SQLSERVER, MYSQL, ORACLE, TERADATA}) // TODO fix postgresql
+    @ExcludeIn({CUBRID, DB2, DERBY, HSQLDB, POSTGRESQL, SQLITE, TERADATA})
```

Comment:
what do you mean? The ExcludeIn usage for this method is the same on the other branch.